modules/videogrid-workflowoperation/src/main/java/org/opencastproject/workflow/handler/videogrid/VideoGridWorkflowOperationHandler.java

```diff
@@ -0,0 +1,1007 @@
+      return height;
+    }
+    public void setHeight(int height) {
+      this.height = height;
+    }
+    public String getName() {
+      return name;
+    }
+    public void setName(String name) {
+      this.name = name;
+    }
+    public String getBgColor() {
+      return bgColor;
+    }
+    public void setBgColor(String bgColor) {
+      this.bgColor = bgColor;
+    }
+
+    LayoutArea(int width, int height) {
+      this.width = width;
+      this.height = height;
+    }
+
+    LayoutArea(String name, int x, int y, int width, int height, String bgColor) {
+      this(width, height);
+      this.name = name;
+      this.x = x;
+      this.y = y;
+      this.bgColor = bgColor;
+    }
+  }
+
+  class VideoInfo
+  {
+    private int aspectRatioWidth = 16;
+    private int aspectRatioHeight = 9;
+    private long timeStamp = 0;
+    private long nextTimeStamp = 0;
+    private long startTime = 0;
+    private long duration = 0;
+    private String filename = "filename.mp4";
+
+    public int getAspectRatioWidth() {
+      return aspectRatioWidth;
+    }
+    public void setAspectRatioWidth(int aspectRatioWidth) {
+      this.aspectRatioWidth = aspectRatioWidth;
+    }
+    public int getAspectRatioHeight() {
+      return aspectRatioHeight;
+    }
+    public void setAspectRatioHeight(int aspectRatioHeight) {
+      this.aspectRatioHeight = aspectRatioHeight;
+    }
+    public long getTimeStamp() {
+      return timeStamp;
+    }
+    public void setTimeStamp(long timeStamp) {
+      this.timeStamp = timeStamp;
+    }
+    public long getNextTimeStamp() {
+      return nextTimeStamp;
+    }
+    public void setNextTimeStamp(long nextTimeStamp) {
+      this.nextTimeStamp = nextTimeStamp;
+    }
+    public long getStartTime() {
+      return startTime;
+    }
+    public void setStartTime(long startTime) {
+      this.startTime = startTime;
+    }
+    public long getDuration() {
+      return duration;
+    }
+    public void setDuration(long duration) {
+      this.duration = duration;
+    }
+    public String getFilename() {
+      return filename;
+    }
+    public void setFilename(String filename) {
+      this.filename = filename;
+    }
+
+    VideoInfo() {
+
+    }
+
+    VideoInfo(int height, int width) {
+      aspectRatioWidth = width;
+      aspectRatioHeight = height;
+    }
+
+    VideoInfo(String filename, long timeStamp, int aspectRatioHeight, int aspectRatioWidth, long startTime)
+    {
+      this(aspectRatioHeight, aspectRatioWidth);
+      this.filename = filename;
+      this.timeStamp = timeStamp;
+      this.startTime = startTime;
+    }
+  }
+
+  class Offset
+  {
+    private int x = 16;
+    private int y = 9;
+
+    public int getX() {
+      return x;
+    }
+    public void setX(int x) {
+      this.x = x;
+    }
+    public int getY() {
+      return y;
+    }
+    public void setY(int y) {
+      this.y = y;
+    }
+
+    Offset(int x, int y) {
+      this.x = x;
+      this.y = y;
+    }
+  }
+
```

Comment:
@Arnei Can you put a comment like for example
```suggestion
// Class Video decision list...   
```

Reply:
For whatever reason the file you're referencing here seems to be part of a previous commit, not the newest one (c90c6f3). In the most recent commit, you should find a comment similar to what you requested. Additionally, the word "webcam" should not appear in any logging statements anymore :)